src/pytocs.sln

```diff
@@ -7,11 +7,15 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Runtime", "..\Runtime\Runtime.csproj", "{D49FD296-62AA-4C97-9265-06201FF76BB6}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Pytocs.Gui", "Gui\Pytocs.Gui.csproj", "{4F1856B8-5E76-4767-AA01-881AB514DD8D}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Pytocs.Gui", "Gui\Pytocs.Gui.csproj", "{4F1856B8-5E76-4767-AA01-881AB514DD8D}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Core", "Core\Core.csproj", "{32899970-0E99-4DEB-A27C-43AAD6299F3F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Pytocs", "Core\Pytocs.csproj", "{32899970-0E99-4DEB-A27C-43AAD6299F3F}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Cli", "Cli\Cli.csproj", "{7C1D87D8-C71A-40FA-97E3-2ACDBECDEE92}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Pytocs.Cli", "Cli\Pytocs.Cli.csproj", "{7C1D87D8-C71A-40FA-97E3-2ACDBECDEE92}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Pytocs.Tests", "Pytocs.Tests\Pytocs.Tests.csproj", "{4A4B8856-2696-4ABF-895C-EB4F1D13EA03}"
```

Comment:
I prefer the name `UnitTests` for projects containing unit tests. Also, I'd rather have the projects named `Cli`, `Gui` rather than `Pytocs.Cli`, `Pytocs.Gui` -- the *.csproj name don't need to contain the full namespace of the code.

Reply:
Will be fixed.